docs/example-bpf.md

```diff
@@ -1,30 +1,35 @@
 # Example BPF Programs
 
-Example applications that use the `bpfd-go` bindings can be found in
-[examples/go-xdp-counter/](https://github.com/bpfd-dev/bpfd/tree/main/examples/go-xdp-counter) and
-[examples/go-tc-counter/](https://github.com/bpfd-dev/bpfd/tree/main/examples/go-tc-counter).
+Example applications that use the `bpfd-go` bindings can be found in the
```

Comment:
So a high level comment... I would break these up into different docs under a docs/examples dir since this one is so long. I know it renders with a table of contents on our website but most users are gonna go "I just want to run this thing FAST how do I do it" So we should make that easy to find.... 

Additionally I'd probably have 3 docs 1: Building From Source 2: Running Locally 3: Running on K8s 
+
maybe even a "Quick Start" that just shows the explicit commands need to run it using prebuilt images and `sudo` all together.

WDYT?

Reply:
So I'll take a look. I think the "Quick Start" is the tutorial.md. I am working on a PR to update that file. That may still need to be simpler. The examples are for the next level, i.e. writing your own program. Not saying they don't need to be simplified, but just a point to consider.

Reply:
Yeah but even in the existing quick start we only show how to load a program.... I think the real value add is much better shown when loading a program that's actually doing something IMO 

I hear you though we could leave these out of the quick start for now.